src/py42/_internal/clients/archive.py

```diff
@@ -10,6 +11,76 @@ def get_data_key_token(self, device_guid):
         data = {u"computerGuid": device_guid}
         return self._session.post(uri, data=json.dumps(data))
 
+    def get_single_archive(self, archive_guid):
+        """Gets single archive information by GUID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            archive_guid (int): The GUID for the archive.
+
+        Returns:
+            :class:`py42.response.Py42Response`: A response containing archive information.
+        """
+        uri = u"/api/Archive/{}".format(archive_guid)
+        return self._session.get(uri)
+
+    def get_page(self, page_num, page_size=None, **kwargs):
+        """Gets an individual page of archives.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            page_num (int): The page number to request.
+            page_size (int, optional): The number of archives to return per page.
+
+        Returns:
+            :class:`py42.response.Py42Response`
+        """
+        uri = u"/api/Archive"
+        page_size = page_size or settings.items_per_page
+        params = dict(
+            pgNum=page_num,
+            pgSize=page_size,
+            **kwargs
+        )
+        return self._session.get(uri, params=params)
+
+    def get_archives_from_value(self, id_value, id_type):
+        """Gets archive information from a query value.
```

Comment:
How about `Gets archives information from an ID, such as a User UID, a Device GUID, or the archive GUID.`

Reply:
Or, in the `args` section, enumerate the supported types

Reply:
Right now, these docs are not publicly facing but they actually might be soon, else I would have just suggested removing them

Reply:
I updated it, but only the initial description as you suggested.